s3upload.js

```diff
@@ -60,7 +60,7 @@ S3Upload.prototype.createCORSRequest = function(method, url) {
 S3Upload.prototype.executeOnSignedUrl = function(file, callback) {
     var xhr = new XMLHttpRequest();
     var fileName = file.name.replace(/\s+/g, "_");
-    xhr.open('GET', this.signingUrl + '?objectName=' + fileName, true);
+    xhr.open('GET', this.signingUrl + '?objectName=' + fileName + '&contentType=' + file.type, true);
```

Comment:
I tried to find compatibility but can't seem to find anything at the moment.  I agree, if `file.type` is undefined maybe we should roll with `mime.lookup`